CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 * [ENHANCEMENT] Compression updates: Added s2, improved snappy performance [#961](https://github.com/grafana/tempo/pull/961) (@joe-elliott)
 * [ENHANCEMENT] Add search block headers [#943](https://github.com/grafana/tempo/pull/943) (@mdisibio)
 * [ENHANCEMENT] Add search block headers for wal blocks [#963](https://github.com/grafana/tempo/pull/963) (@mdisibio)
+* [ENHANCEMENT] Add received bytes metric in ingesters [#979](https://github.com/grafana/tempo/pull/979) (@mapno)
```

Comment:
Please put the new metric name in the changelog.

Reply:
Added, thanks